msal/application.py

```diff
@@ -151,6 +151,14 @@ def __init__(
             "The provided signature value did not match the expected signature value",
             you may try use only the leaf cert (in PEM/str format) instead.
 
+            **NEW**
+            it can also be a completly pre-signed JWT that you've assembled yourself
+            simply pass a container containing only the key "jwt", like this:
+
+                {
+                    "jwt": "..."
```

Comment:
Our API surface would prefer to use a name referring to the intention rather than the data format. Besides, MSAL .Net already supports [confidential client authenticated by a signed assertion](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/Client-Assertions#signed-assertions), with the API `.WithClientAssertion(...)`. So, we would suggest to change the parameter name of this feature from `jwt` to `client_assertion`. Would you be able to change that here and in those test cases in this PR?

Reply:
done.